src/Microsoft.TestPlatform.Common/DataCollection/AfterTestRunEndResult.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Common.DataCollection
+{
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Payload object that is used to exchange data between datacollector process and runner process.
+    /// </summary>
+    [DataContract]
+    public class AfterTestRunEndResult
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AfterTestRunEndResult"/> class.
+        /// </summary>
+        /// <param name="environmentVariables">
+        /// The collection of attachment sets.
+        /// </param>
+        /// <param name="dataCollectionEventsPort">
```

Comment:
The description here needs to be updated.

Reply:
fixed